gapis/api/vulkan/api/command_buffer_control.api

```diff
@@ -619,13 +637,23 @@ sub void resetBufferCommands(ref!CommandBufferObject obj) {
   clear(obj.BufferCommands.vkCmdDispatchBase)
   clear(obj.BufferCommands.vkCmdSetLineStippleEXT)
 
+  // Vulkan 1.2
+  clear(obj.BufferCommands.vkCmdBeginRenderPass2)
+  clear(obj.BufferCommands.vkCmdEndRenderPass2)
+  clear(obj.BufferCommands.vkCmdNextSubpass2)
+
   // @extension("VK_EXT_transform_feedback")
   clear(obj.BufferCommands.vkCmdBindTransformFeedbackBuffersEXT)
   clear(obj.BufferCommands.vkCmdBeginTransformFeedbackEXT)
   clear(obj.BufferCommands.vkCmdEndTransformFeedbackEXT)
   clear(obj.BufferCommands.vkCmdBeginQueryIndexedEXT)
   clear(obj.BufferCommands.vkCmdEndQueryIndexedEXT)
   clear(obj.BufferCommands.vkCmdDrawIndirectByteCountEXT)
+
+  // @extension("VK_KHR_create_renderpass2")
```

Comment:
Isn't vkCreateRenderPass2 also supposed to be included here?

Reply:
No, because VkCreateRenderPass2(KHR) is not a buffer command.

Buffer command means "a command that is recorded to a command buffer, then executed when that buffer is submitted to a queue". Which in practice means the commands that starts with `VkCmd....` are the buffer commands.